utils/text/operations_test.cpp

```diff
@@ -90,6 +91,8 @@ ATF_TEST_CASE_BODY(escape_xml__some_escaping)
 
     ATF_REQUIRE_EQ("&quot;&amp;&lt;&gt;&apos;", text::escape_xml("\"&<>'"));
     ATF_REQUIRE_EQ("&amp;&amp;&amp;", text::escape_xml("&&&"));
+    ATF_REQUIRE_EQ("&amp;#8;&amp;#11;", text::escape_xml("\b\v"));
+    ATF_REQUIRE_EQ("\t&amp;#127;BAR&amp;", text::escape_xml("\t\x7f""BAR&"));
```

Comment:
What's the point of the double "" in the middle of the argument passed to `escape_xml`? Seems unnecessary.